crates/artifacts/zksolc/src/error.rs

```diff
@@ -91,6 +84,7 @@ impl fmt::Display for Error {
         }) {
             let _ = lines.next();
         }
+        */
```

Comment:
We probably don't want commented code

Reply:
Maybe but I would like to take some time after we merge to check error messages on both compilers and see if it makes sense in all cases to not have this because I arrived at removing this through mere trial and error. If so I'll delete then.